pkg/controller/datavolume-controller.go

```diff
@@ -2824,13 +2865,24 @@ func (r *DatavolumeReconciler) detachPvcDeleteDv(pvc *corev1.PersistentVolumeCla
 
 func updatePvcOwnerRefs(pvc *corev1.PersistentVolumeClaim, dv *cdiv1.DataVolume) {
```

Comment:
This requires a unit test, and that will help us in the future if we ever need to change or refactor the code here.

Reply:
Sure, adding now.